### ISSUE TITLE
add possibility to read TGraph from `.C` file

### ISSUE THREAD

**Issue by clelange** [HEPData/hepdata_lib#8]
I have some example code that one can build on.

**Comment by AndreasAlbert** [comment]
Exists here: https://github.com/clelange/hepdata_lib/blob/master/hepdata_lib/__init__.py#L413-L416

**Comment by clelange** [comment]
Excellent! I just realised it's actually reading the TGraph from a `.C` file - will rename the issue.

Here is some code:

```
rx = readGraph(infile,"Graph0_fx1")
ry = readGraph(infile,"Graph0_fy1")
d1 = makeDict(rx,ry)
predown,preup,x = makeLists(d1)
```
which uses:
```
def readGraph(infile,graphname):
    f = open(infile,"r")
    r=[]
    start=0
    for line in f.readlines():
        if '#' in line: continue
        if (graphname in line) and ("{" in line):
            start=1
            continue
        if "}" in line:
            if start==1: r.append(float(line.split("}")[0]) )
            start=0
        if start==1:
            r.append(float(line.split(",")[0]) )
    return r
  
def getErrors(down,up,nominal):
    i=0
    errup =[]
    errdown=[]
    for n in nominal:
        errdown.append(down[i]-n)
        errup.append(up[i]-n)
        print(str(n) + " +- "+ str(errup[-1])+" "+str(errdown[-1]))
        i+=1
    return errdown,errup
         
def makeDict(x,y):
    d ={}
    ati=0
    for i in x:
        d[i]=[]
    for i in range(0,len(x)):    
        d[x[i]].append(y[i])
    return d
        
def makeLists(dic):
    up=[]
    down=[]
    keys = sorted(dic.keys())
    for k in keys:
        for i in range(0,len(dic[k])):
            if i ==0:
                up.append(dic[k][0])
            if i ==1:
                down.append(dic[k][1])
    return up,down,keys
```

**Comment by clelange** [comment]
Closed by #135
